test/util.ts

```diff
@@ -6,3 +6,31 @@ export function expectFeeEstimationStructure(fee: any) {
     expect(typeof fee.amount).to.equal("bigint");
     expect(fee.unit).to.equal("wei");
 }
+
+export function ensureEnvVar(varName: string): string {
+    if (!process.env[varName]) {
+        throw new Error(`Env var ${varName} not set or empty`);
+    }
+    return <string>process.env[varName];
+}
+
+
+/**
+ * Receives a hex address, converts it to bigint, converts it back to hex.
+ * This is done to strip leading zeros.
+ * @param address a hex string representation of an address
+ * @returns an adapted hex string representation of the address
+ */
+ function adaptAddress(address: string) {
+    return "0x" + BigInt(address).toString(16);
+  }
+  
+  /**
+   * Expects address equality after adapting them.
+   * @param actual 
+   * @param expected 
+   */
+  export function expectAddressEquality(actual: string, expected: string) {
+    expect(adaptAddress(actual)).to.equal(adaptAddress(expected));
+  }
+  
```

Comment:
Missing EOL here

Reply:
It was actually reporting extra whitespace; fixed it: https://github.com/Shard-Labs/starknet-hardhat-example/pull/65/commits/8788e4f3845fbc8e5a1923a7d28def79fad9c5d0